vue.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
   devServer: {
     proxy: {
       '/graphql': {
-        target: 'http://127.0.0.1:12800',
+        target: `${process.env.SW_PROXY_TARGET || "http://127.0.0.1:12800"}`,
```

Comment:
Add environment variables to prevent others from accidentally committing after modifying this at development time.

Reply:
FYI @Fine0830 @TinyAllen